### PR TITLE
[Merged by Bors] - First round of migration of integration-tests

### PR DESCRIPTION
**Pull request by sbernauer**
## Description

Well that was easier than expected.
LDAP tests will follow
Run with `scripts/run_tests.sh --parallel 4`

## Review Checklist
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added (or not applicable)
- [ ] Documentation added (or not applicable)
- [ ] Changelog updated (or not applicable)
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
- [ ] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: tests/templates/kuttl/druid-connection/01-install-superset.yaml.j2 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+spec:
+  version: {{ test_scenario['values']['superset'] }}
+  statsdExporterVersion: v0.22.4
+  credentialsSecret: superset-credentials
+  loadExamplesOnInit: false
+  nodes:
+    roleGroups:
+      default:
+        replicas: 1
+        config: {}
```
Comment (adwk67) [inline tests/templates/kuttl/druid-connection/01-install-superset.yaml.j2:35]: Optional: this can be removed:
```suggestion
```

**Review by adwk67 — APPROVED** [review]
Tests all OK (with local operator and also version loaded via stackablectl).
Approved. Just suggested two optional changes.

**Comment by sbernauer** [comment]
thanks a lot!

**Pull request closed without merging**